modules/services/utils/src/main/java/au/com/shiftyjelly/pocketcasts/utils/featureflag/Feature.kt

```diff
@@ -52,9 +52,26 @@ enum class Feature(
         hasDevToggle = true,
     );
 
+    fun isCurrentlyExclusiveToPatron(): Boolean {
+        val isReleaseCandidate = ReleaseVersion.currentReleaseVersion.releaseCandidate != null
+        val relativeToEarlyAccessState = (this.tier as? FeatureTier.Plus)?.patronExclusiveAccessRelease?.let {
+            ReleaseVersion.currentReleaseVersion.comparedToEarlyPatronAccess(it)
+        }
+        return when (relativeToEarlyAccessState) {
+            null -> false
+            EarlyAccessState.Before,
+            EarlyAccessState.During -> !isReleaseCandidate
+            EarlyAccessState.After -> false
+        }
+    }
+
     companion object {
 
-        fun isAvailable(feature: Feature, userTier: UserTier) = when (userTier) {
+        fun isAvailable(
```

Comment:
>  I feel like having both this Feature::isAvailable method and the FeatureFlag::isEnabled methods is a bit confusing since they sound so similar. What would you think about renaming this method to something like isUserEntitled

Same as above,  I renamed this method from `isAvailable` to `isUserEntitled` in https://github.com/Automattic/pocket-casts-android/commit/e1f217d687f87bd4031f97ad4ee898affd0be1da. 

> Also, what would you think about moving this method to the FeatureFlag object so it's at the same place as the isEnabled logic? I don't feel strongly, just a thought.

I too don't feel strongly but we can address it in another PR if our opinions change in the future.